### PR TITLE
2184 - Fileupload backspace triggers back in IE11 [v4.19.x]

### LINKED ISSUE #2184: File Upload: On Windows IE, pressing backspace on file upload is causing the browser to go to previous page
**Describe the bug**
File Upload: On Windows IE, pressing backspace on file upload is causing the browser to go to previous page

**To Reproduce**
1. Open Windows Internet Explorer or Edge
2. Go to http://master-enterprise.demo.design.infor.com/components/fileupload/example-index
3. Select a file on the file upload
4. While on the file upload, press backspace
5. See issue, file upload will be cleared but will also cause the browser to go to the previous page

**Expected behavior**
File upload field should be cleared and stay in the page when backspace is pressed.

**Version**
- ids-enterprise: [e.g. v4.7.0 - current version]

**Screenshots**
No applicable screenshot.

**Platform**
 - OS Version: Windows 10
 - Browser Name: Internet Explorer and Edge

**Additional context**
No additional context.

### PR DESCRIPTION
**Pull request by davidcarlsonberg**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Clicking backspace to remove the uploaded causes the browser to go back to the previous page. This PR prevents that default behavior when clearing the file upload input.

**Related github/jira issue (required)**:
Closes #2184.

**Steps necessary to review your pull request (required)**:
- Pull branch, run app
- in IE11
  - Visit http://localhost:4000/components/fileupload/example-index
  - Click to upload a file
  - Once the file is in the input, focus the input, and attempt backspace to remove the file
  - The file should be removed

~- [ ] An e2e or functional test for the bug or feature.~
- [x] A note to the change log.

### CODE REVIEW

File: src/components/fileupload/fileupload.js [hunk 1]
```diff
@@ -97,6 +98,7 @@ FileUpload.prototype = {
         elem.parent().find('[type="file"]').trigger('click');
         handle = true;
       } else if (e.which === 8) {
+        if (env.browser.isIE11()) e.preventDefault();
```
Comment (tmcconechy) [inline src/components/fileupload/fileupload.js]:
```suggestion
        if (env.browser.isIE11()) {
          e.preventDefault();
        }
```

Comment (tmcconechy) [reply src/components/fileupload/fileupload.js]: Thought this would generate a lint error?

Comment (davidcarlsonberg) [reply src/components/fileupload/fileupload.js]: Looks like not, but is this a standard to which we'd like to adhere?

Comment (tmcconechy) [reply src/components/fileupload/fileupload.js]: Maybe was in the old rules. Actually doesn’t matter to me. I think ignore me then.

**Comment by davidcarlsonberg** [comment]
Needs merge of #2269 to pass locale tests.

**Pull request merged**